extensions/amp-analytics/linker-id-receiving.md

```diff
@@ -98,6 +98,45 @@ Two macros are supported for the `value` field. [`QUERY_PARAM`](https://github.c
 
 If there's error resoving the value, or the value is resolved to empty string. Nothing will be written to the cookie.
 
+##### SameSite Cookies
+
+To specify a SameSite value for a cookie, include a key-value pair where the key is `sameSite` and the value is one of `Strict`, `Lax` or `None`. If `sameSite` is not specified or is an invalid value, it defaults to `Lax` (which is the current [browser behavior](https://web.dev/samesite-cookies-explained/#samesitelax-by-default)). If `sameSite` is set to `None`, we pair it with the `Secure` attribute as [recommended](https://web.dev/samesite-cookies-explained/#samesitenone-must-be-secure) to prevent the cookie from getting rejected.
```

Comment:
Update the default to `LAX` description.

Reply:
Done.